.github/dependabot.yml

```
@@ -4,4 +4,15 @@ updates:
   - package-ecosystem: "gomod"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "daily"
+    ignore:
+      - dependency-name: "github.com/edgexfoundry/go-mod-core-contracts/v4"
+        # Included when device-sdk-go is updated
+      - dependency-name: "github.com/edgexfoundry/go-mod-messaging/v4"
+        # Included when device-sdk-go is updated
+      - dependency-name: "github.com/edgexfoundry/go-mod-registry/v4"
+        # Included when device-sdk-go is updated
+      - dependency-name: "github.com/edgexfoundry/go-mod-secrets/v4"
+        # Included when device-sdk-go is updated
+      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/v4"
```

```suggestion
      - dependency-name: "github.com/edgexfoundry/go-mod-bootstrap/*"
```
use wildcard to save the future version upgrade